akirachix/Kuwala-Frontend
# Feature/overview

# Engineers Checklist
Dependencies
APIs for metrics, including /api/users and /api/fooditems endpoints.

## What this PR does?
This pull request displays the metrics for the dashboard overview on the admin panel of the DishHub application. It addresses the need to display key performance indicators such total food items and total users using data fetched from the API. 
Perhaps answering the following questions will help create a good description:
* What is this?
The dashboard is a crucial part of the admin's interaction with the system,providing real time data for the admin.
* How are you doing this? 
Implementing API calls to fetch metrics and display them in visual components like bar charts .
* 
- List of updates for features, enhancements, or bugfixes

## Type of change

Specify the kind of PR/MR

Please delete options that are not relevant.


## How can it be tested?
Run npm install to ensure all dependencies are in place.
Launch the app locally using npm run dev.
- If web or mobile, screenshots will be very helpf
![Screenshot from 2024-10-02 06-49-06](https://github.com/user-attachments/assets/ae802bb9-177c-473e-b425-898aaedb878f)




# Where is it documented?
- List JIRA tickets
